src/IERC20.sol

```diff
@@ -0,0 +1,98 @@
+     */
+    function symbol() external view returns (string memory);
+
+    /**
+     * @dev Returns the decimal precision used by the token.
+     */
+    function decimals() external view returns (uint8);
+
+    /**
+     * @dev Returns the total amount of tokens in existence.
+     */
+    function totalSupply() external view returns (uint256);
+
+    /**
+     * @dev   Returns the amount of tokens owned by a given account.
+     * @param account Account that owns the tokens.
+     */
+    function balanceOf(address account) external view returns (uint256);
+
+    /**
+     * @dev   Function that returns the allowance that one account has given another over their tokens.
+     * @param owner   Account that tokens are approved from.
+     * @param spender Account that tokens are approved for.
+     */
+    function allowance(address owner, address spender) external view returns (uint256);
+
+    /**
+     * @dev   Function that allows one account to set the allowance of another account over their tokens.
+     * @dev   Emits an {Approval} event.
+     * @param spender Account that tokens are approved for.
+     * @param amount  Amount of tokens that have been approved.
+     */
+    function approve(address spender, uint256 amount) external returns (bool);
+
+    /**
+     * @dev   Function that allows one account to increase the allowance of another account over their tokens by a given amount.
+     * @dev   Emits an {Approval} event.
+     * @param spender Account that tokens are approved for.
+     * @param amount  Incremental amount of tokens that have been approved.
+     */
+    function increaseAllowance(address spender, uint256 amount) external returns (bool);
+
+    /**
+     * @dev   Function that allows one account to decrease the allowance of another account over their tokens by a given amount.
+     * @dev   Emits an {Approval} event.
+     * @param spender Account that tokens are approved for.
+     * @param amount  Decremental amount of tokens that have been approved.
+     */
+    function decreaseAllowance(address spender, uint256 amount) external returns (bool);
+
+    /**
+     * @dev   Moves an amount of tokens from `msg.sender` to a specified account.
+     * @dev   Emits a {Transfer} event.
+     * @param recipient Account that recieves tokens.
+     * @param amount    Amount of tokens that are transferred.
+     * @return          Boolean amount indicating whether the operation succeeded.
+     */
+    function transfer(address recipient, uint256 amount) external returns (bool);
+
+    /**
+     * @dev   Moves a pre-approved amount of tokens from a sender to a specified account.
+     * @dev   Emits a {Transfer} event.
+     * @param owner     Account that tokens are moving from.
+     * @param recipient Account that recieves tokens.
+     * @param amount    Amount of tokens that are transferred.
+     */
+    function transferFrom(address owner, address recipient, uint256 amount) external returns (bool);
+
+}
```

Comment:
These are all following semantic order, not alphabetical. I honestly think this way it makes more sense, but our style guide asks for alphabetical, right?